.github/workflows/nightly-simulator.yml

```
@@ -0,0 +1,388 @@
+          fi
+          echo "master_seed=${master_seed}" >> "${GITHUB_OUTPUT}"
+
+          if [ "${rc}" -eq 0 ]; then
+            echo "status=passed" >> "${GITHUB_OUTPUT}"
+            # Even on green we always upload an artifact (per the
+            # issue body: "One nightly sweep run completes and
+            # uploads its artifact even on green"). Write a
+            # green-status marker so the artifact is non-empty.
+            cat > "${VIBIX_SIM_FAULT_DIR}/SUMMARY.txt" <<EOF
+nightly-simulator: GREEN
+master_seed=${master_seed}
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)